src/app/beer_garden/config.py

```diff
@@ -723,6 +723,95 @@ def _parse_args(args: Sequence[str]) -> Tuple[YapconfSpec, dict]:
     },
 }
 
+_PARENT_SPEC = {
+    "type": "dict",
+    "items": {
+        "http": {
+            "type": "dict",
+            "items": {
+                "enable": {
+                    "type": "bool",
+                    "default": False,
+                    "description": "Publish events to parent garden over HTTP",
+                    "alt_env_names": ["PARENT_HTTP"],
+                },
+                "ssl": {
+                    "type": "dict",
+                    "items": {
+                        "enabled": {
+                            "type": "bool",
+                            "default": False,
+                            "description": "Serve content using SSL",
+                            "cli_separator": "_",
+                        },
+                        "private_key": {
+                            "type": "str",
+                            "description": "Path to a private key",
+                            "required": False,
+                        },
+                        "public_key": {
+                            "type": "str",
+                            "description": "Path to a public key",
+                            "required": False,
+                        },
+                        "ca_cert": {
+                            "type": "str",
+                            "description": (
+                                "Path to CA certificate file to use for SSLContext"
+                            ),
+                            "required": False,
+                        },
+                        "ca_path": {
+                            "type": "str",
+                            "description": (
+                                "Path to CA certificate path to use for SSLContext"
+                            ),
+                            "required": False,
+                        },
+                        "client_cert_verify": {
+                            "type": "str",
+                            "description": (
+                                "Client certificate mode to use when handling requests"
+                            ),
+                            "choices": ["NONE", "OPTIONAL", "REQUIRED"],
+                            "default": "NONE",
+                        },
+                    },
+                },
+                "port": {
+                    "type": "int",
+                    "default": 2337,
+                    "description": "Serve content on this port",
+                },
+                "url_prefix": {
+                    "type": "str",
+                    "default": "/",
+                    "description": "URL path prefix",
+                    "required": False,
+                },
+                "host": {
+                    "type": "str",
+                    "default": "0.0.0.0",
+                    "description": "Host for the HTTP Server to bind to",
+                    "alt_env_names": ["PARENT_HOST"],
```

Comment:
This alt name should be removed - it's setting us up for problems whenever we add additional transport mechanisms. If you need to set this from compose just use the normal name `PARENT_HTTP_HOST`.